packages/temen/test/is.test.js

```diff
@@ -19,3 +19,22 @@ describe('is', () => {
     expect(isBoolean(true)).toBe(true);
   });
 });
+
+describe('isNil 함수는 value가 null 또는 undefined 인지 확인합니다.', () => {
+  test(`isNil은 value가 null이면 true 를 반환한다. `, () => {
+    expect(isNil(null)).toBeTruthy();
+  });
+  test(`isNil은 value가 (void 0)이면 true 를 반환한다.`, () => {
+    expect(isNil(void 0)).toBeTruthy();
+  });
+
+  test(`isNil은 value가 NaN이면 false 를 반환한다.`, () => {
+    expect(isNil(NaN)).toBeFalsy();
+  });
+  test(`isNil은 value가 ''이면 false 를 반환한다.`, () => {
+    expect(isNil('')).toBeFalsy();
+  });
+  test(`isNil은 value가 undefined 이면 true 를 반환한다.`, () => {
+    expect(isNil(undefined)).toBeTruthy;
```

Comment:
```suggestion
    expect(isNil(undefined)).toBeTruthy();
```

함수 호출이 되어야 할 것 같습니다!

Reply:
앗 👀